nixos/modules/services/web-servers/minio.nix

```diff
@@ -29,23 +35,18 @@ in
       description = "The config directory, for the access keys and other settings.";
     };
 
-    accessKey = mkOption {
-      default = "";
+    rootUser = mkOption {
       type = types.str;
       description = ''
-        Access key of 5 to 20 characters in length that clients use to access the server.
-        This overrides the access key that is generated by minio on first startup and stored inside the
-        <literal>configDir</literal> directory.
```

Comment:
Why did you removed all of that useful documentation?

Reply:
Because it is no longer true.

Reply:
That's fair enough.